docs/ide-usage.md

```diff
@@ -68,8 +68,10 @@ The following plugins are needed:
      * HTML: `Web → HTML Files → Editor → Indent using spaces`.
      * CSS: `Web → CSS Files → Editor → Indent using spaces`.
      * XML: `XML → XML Files → Editor → Indent using spaces`.
-   * Validation: We do not validate HTML, JSP, and XML.<br>
-     `Validation` → uncheck the `Build` option for `HTML Syntax Validator`, `JSP Content Validator`, `JSP Syntax Validator`, and `XML Validator`.
+   * Validation: 
+     * We do not validate HTML, JSP, and XML.`Validation` → uncheck the `Build` option for `HTML Syntax Validator`, `JSP Content Validator`, `JSP Syntax Validator`, and `XML Validator`.
+     * Disable JavaScript validation for node_modules folder. `Validation` → click the `...` settings button for `JavaScript Validation` → if `Exclude Group` is not already in the list then click `Add Exclude Group...` → `Exclude Group` → `Add Rule...` → select `Folder or file name` → `Next` → `Browse Folder...` → navigate to the `node_modules` folder → `OK` → then click on `Finish`.
```

Comment:
@Yogitha6 I feel the word `select` before `Folder or file name` is unnecessary as it's already implied; similarly for the phrase `then click on` before `Finish`. It'll be cleaner if they were removed.

Also, the `OK` step can be combined with the `navigate...` step, as it's platform-specific (on macOS the file browser shows `Open` instead of `OK`). This would be cleaner:
... → `Browse Folder...` → navigate to the `node_modules` folder and confirm → `Finish`.